skhep/math/geometry.py

```diff
@@ -608,22 +666,68 @@ def distance ( self , other ) :
         >>> print line.distance ( point )
         >>> print line.distance ( plane )        
         """
-        if   isinstance ( other , Point3D ) :
```

Comment:
On line 654 the example should read print(plane1!=plane2). Notice the 2 changes, the second being the most important one.